docs/stdlib/doc/fs.md

```diff
@@ -0,0 +1,81 @@
+## `change_owner`
+```ab
+pub fun change_owner(user: Text, path: Text): Bool 
+```
+
+Changes the owner of a file.
+
+If the file doesn't exist, it returns `false`
+
+
+## `create_dir`
+```ab
+pub fun create_dir(path: Text): Null 
+```
+
+Creates a directory with all parent directories as required.
+
+
+## `create_symbolic_link`
+```ab
+pub fun create_symbolic_link(origin: Text, destination: Text): Bool 
+```
+
+Creates a symbolic link.
+
+If the file doesn't exist, it returns a boolean and prints a message.
+
+
+## `dir_exist`
+```ab
+pub fun dir_exist(path) 
+```
+
+Checks if a directory exists.
```

Comment:
how does it do that? what if theres a file with the same path? what type is `path`? what does it return?